tests/testthat/test_IntegralScores.R

```diff
@@ -0,0 +1,123 @@
+context("Test integral scores")
+
+test_that("Expected sample size is computed correctly",{
```

Comment:
very difficult to follow - could you maybe comment a bit on what you ACTUALLY want to do?